mono/utils/mono-state.c

```diff
@@ -687,6 +692,7 @@ mono_native_state_add_thread (MonoStateWriter *writer, MonoThreadSummary *thread
 static void
 mono_native_state_add_ee_info  (MonoStateWriter *writer)
 {
+#ifndef MONO_PRIVATE_CRASHES
```

Comment:
Where does this conditional come from?  This particular change also seems to modify behavior, so it's more than just fixing compiler warnings.

Reply:
As far as I can tell it comes from configure, controlled with --with-crash_privacy=yes,no. What behavior does this change modify? It turns existing function to empty body, and changes to call it unconditionally, to fix unused function warning.

Reply:
this is fine